packages/office-addin-debugging/src/commands.ts

```diff
@@ -7,6 +7,47 @@ import * as devSettings from "office-addin-dev-settings";
 import { AppType, parseAppType, parseDebuggingMethod, startDebugging } from "./start";
 import { stopDebugging } from "./stop";
 
+function getManifestPath(manifestPath: string, appTypeToDebug: string | undefined, prod: boolean): string {
```

Comment:
It would be really useful to have some comments explaining the logic of this function. Like - under what conditions do we modify the manifestPath and how. Can you add that?